src/draft-06-schema.json

```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
whats the point of having this file in the repo? also i see that the schemas in this repo are using draft 07 instead of 06?

Reply:
If needed for ah... This version only embedded json schema v7